src/function/string/operations/include/lpad_operation.h

```diff
@@ -3,33 +3,32 @@
 #include <cassert>
 #include <cstring>
 
-#include "pad_operation.h"
+#include "base_pad_operation.h"
 
 #include "src/common/types/include/ku_string.h"
 
 using namespace std;
-using namespace kuzu::common;
+using namespace kuzu::function::operation;
 
 namespace kuzu {
 namespace function {
 namespace operation {
 
-struct Lpad : PadOperation {
+struct Lpad : BasePadOperation {
```

Comment:
lpad will cause an assertion failure if the count is a negative number:
```
return lpad('5', -1, '>')
```

Reply:
changed this to same as duckdb behaviour, they change -ve count to 0